openstack/compute/v2/extensions/availabilityzones/results.go

```diff
@@ -1,12 +1,8 @@
 package availabilityzones
 
-// ServerExt is an extension to the base Server object
-type ServerExt struct {
+// ServerAvailabilityZoneExt is an extension to the base Server result which
+// includes the Availability Zone information.
+type ServerAvailabilityZoneExt struct {
```

Comment:
I'm OK with renaming these, but this is a mouthful, especially when it's used in user code. Here, it seems (to me) like `availabilityzone.Ext` is clear. I'd like to hear suggestions, with an eye toward keeping it succinct yet clear when used in user code.